Note: Tenants with 3rd party MX on Effectiveness (#2956)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mail-flow-best-practices/manage-mail-flow-using-third-party-cloud.md

## Using a third-party cloud service with Microsoft 365 or Office 365

### Scenario 1 - MX record points to third-party spam filtering

>   > [!IMPORTANT]

Reviewer: ```suggestion

    > [!IMPORTANT]
```

>   > Microsoft strongly recommends you to enable Enhanced Filtering for Connectors or bypass filtering completely using a mail flow rule (check out point 5). Failure to follow this will inevitably result in misclassification of inbound email to your organization and to subpar experience of Office 365 email and protection features.

Reviewer: ```suggestion
  > We strongly recommend that you enable Enhanced Filtering for Connectors or bypass filtering completely using a mail flow rule (see point 5). Failure to do this will inevitably result in misclassification of inbound emails to your organization and to the subpar experience of Office 365 email and protection features.
```
